CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # CHANGELOG
 
+## Version 3.7.0 GA (01/22/2025)
+
+### Enhancements
+
+* Transition to azure-monitor-opentelemetry-autoconfigure 1.0.0-beta.1, in place of azure-monitor-opentelemetry-exporter [#4037](https://github.com/microsoft/ApplicationInsights-Java/pull/4037)
+* Support for Live Metrics Filtering [#43040](https://github.com/Azure/azure-sdk-for-java/pull/43040), [#43184](https://github.com/Azure/azure-sdk-for-java/pull/43184), [#43289](https://github.com/Azure/azure-sdk-for-java/pull/43289), [#43323](https://github.com/Azure/azure-sdk-for-java/pull/43323), [#43506](https://github.com/Azure/azure-sdk-for-java/pull/43506), [#43744](https://github.com/Azure/azure-sdk-for-java/pull/43744)
```

Comment:
can you add parentheses around the others too, looks like that's been our convention below, I'm not sure if our release automation cares or not about it when formatting the release notes
```suggestion
* Support for Live Metrics Filtering ([#43040](https://github.com/Azure/azure-sdk-for-java/pull/43040), [#43184](https://github.com/Azure/azure-sdk-for-java/pull/43184), [#43289](https://github.com/Azure/azure-sdk-for-java/pull/43289), [#43323](https://github.com/Azure/azure-sdk-for-java/pull/43323), [#43506](https://github.com/Azure/azure-sdk-for-java/pull/43506), [#43744](https://github.com/Azure/azure-sdk-for-java/pull/43744))
```